ktx2Decoder/src/Misc/dataReader.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * Utility class for reading from a data buffer
+ */
+export class DataReader {
+    /**
+     * The current byte offset from the beginning of the data buffer.
+     */
+    public get byteOffset() {
+        return this._dataByteOffset;
+    }
+
+    /**
+     * Indicates the endianness of the data in the buffer
+     */
+    public littleEndian = true;

```

Comment:
`DataReader` class is an helper class that could be used for any purpose (I copied it from the Babylon core).

However in this case I guess it will only be used to read the ktx file which is little endian, so I guess I can hard code the value.

Reply:
It's not a big deal either way. I wrote the original one in Babylon core and it doesn't not have this property.